### PR TITLE
cache coverages

### PR DESCRIPTION
**Pull request by ryan-williams**
removed this during CR process previously but that was a mistake; partitioning debug stats end up performing this whole computation a second time

---
This change is [Reviewable](https://reviewable.io/reviews/hammerlab/guacamole/545)

### CODE REVIEW

File: src/main/scala/org/hammerlab/guacamole/readsets/rdd/RegionRDD.scala [hunk 1]
```diff
@@ -26,19 +27,23 @@ class RegionRDD[R <: ReferenceRegion: ClassTag](@transient rdd: RDD[R])
   def coverage(halfWindowSize: Int, loci: LociSet): RDD[(Position, Coverage)] =
     coverage(halfWindowSize, sc.broadcast(loci))
 
+  val _coverages = mutable.HashMap[(Int, LociSet), RDD[(Position, Coverage)]]()
```
Comment (timodonnell) [inline src/main/scala/org/hammerlab/guacamole/readsets/rdd/RegionRDD.scala]: `coveragesCache` or similar might be a better name

Comment (timodonnell) [inline src/main/scala/org/hammerlab/guacamole/readsets/rdd/RegionRDD.scala]: the cache can also be private, I'd imagine

**Comment by timodonnell** [comment]
LGTM

**Comment by timodonnell** [comment]
oops didn't mean to close that, LGTM

File: src/main/scala/org/hammerlab/guacamole/readsets/rdd/RegionRDD.scala (hunk 1 above)
Comment (ryan-williams) [inline src/main/scala/org/hammerlab/guacamole/readsets/rdd/RegionRDD.scala]: good points, both done

**Pull request merged**